docs/userGuide/plugins/web3Form.md

```diff
@@ -0,0 +1,89 @@
+### Plugin: Web3Forms 
+
+This plugin allows you to create forms whose response will be sent directly to your email, using the [Web3Form](https://web3forms.com/) API. 
+
+To set it up, get an access key from [Web3Forms](https://web3forms.com/). Then add`web3Form` to your site's plugin, and add the `accessKey` parameter via the `pluginsContext`.
```

Comment:
@yucheng11122017 I assume this access key is not a 'secret' key?

Reply:
Urm not exactly? It's linked to the user's email. web3forms uses it to redirect the form submissions to the user's email.

Reply:
Is there potential for misuse of the key? My current understanding is that if someone wants to abuse this key, they could use it to direct the submissions from a different form to the user's email address, but this would have the same effect as simply spamming junk submissions to the original form so it doesn't give a malicious user any additional tool for abuse. 
Unless the key can be used to reveal the user's actual email (which the user might have intended to keep private)?

Reply:
Should be fine, if the key is not meant to be a 'secret' key that should not be made public.